cg/services/order_validation_service/workflows/fastq/validation_service.py

```diff
@@ -46,5 +44,10 @@ def _get_errors(self, raw_order: dict) -> ValidationErrors:
             order_errors=order_errors,
         )
 
-    def parse_and_validate(self, raw_order: dict) -> Order:
-        pass
+    def parse_and_validate(self, raw_order: dict) -> FastqOrder | dict:
+        parsed_order, errors = ModelValidator.validate(order=raw_order, model=FastqOrder)
+        if parsed_order:
+            errors: ValidationErrors = self._perform_rule_validation(order=parsed_order)
+        if not errors.is_empty:
+            return create_order_validation_response(raw_order=raw_order, errors=errors)
```

Comment:
A bit uncertain of what should happen here - should an error be raised or what do we want the FE to look like?